NN_maps.js

```diff
@@ -1,29 +1,33 @@
 /**
-    Name: NerthusMaps
-    Zawiera zmiane map na nocne oraz na sezonowe.
-**/
-try{
+ Name: NerthusMaps
+ Zawiera zmiane map na specjalne czy też sezonowe.
+ **/
+try {
 
 nerthus.maps = {}
 
-nerthus.maps.seasonMaps = function()
+nerthus.maps.customMaps = function ()
 {
-    var season = nerthus.season()
-    for(i in nerthus.mapsArr)
-        if( nerthus.mapsArr[i][0] == season && nerthus.mapsArr[i][1] == map.id )
+    let season = nerthus.season()
+    for (const i in nerthus.mapsArr)
+    {
+        if (nerthus.mapsArr[i][1] === map.id && (nerthus.mapsArr[i][0] === 0 || nerthus.mapsArr[i][0] === season))
+        {
             nerthus.maps.change(nerthus.mapsArr[i][2])
+            break
```

Comment:
Po co ten break? już lepiej tu dać returna a jeszcze lepiej zrobić finda na mapsArr jak już tu grzebiesz ;)

Reply:
[Array.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) nie jest wspierany przez IE, a wspieramy chyba jeszcze jego ostatnią wersję ( #32 )
break przerywa for-a, więc jest równoważny w tym miejscu z returnem. Ale mogę zmienić.

Reply:
Będę musiał poszukać jak się puszcza testy na przeglądarce, mocha gdzieś miała do tego wsparcie, bo teraz to w sumie nie wiem co na czym działa  :/

Reply:
W międzyczasie możesz spullować ;)
A to że dana rzecz nie jest wspierana można po prostu w większości przypadków wyczytać.